README.md

```diff
@@ -1,6 +1,6 @@
 ## Türkçe Kaynaklar Listesi [![Build Status](https://img.shields.io/travis/fatihacet/turkcekaynaklar-com.svg?style=flat-square)](https://travis-ci.org/fatihacet/turkcekaynaklar-com)
 
-Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
+Yazılım geliştirme ve Tasarım(web ve mobil) ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
```

Comment:
Öncelikle yorumlarınız/düzeltmeleriniz için teşekkür ederim.

İlk olarak "Tasarım" kelimesindeki T'yi küçük yazdığım ve boşluk bırakmadığım gözümden kaçmış kusura bakmayın, düzelttiğiniz içinde teşekkür ederim.

Readme dosyasını incelediğimde her bölüm arasında iki boşluk vardı, bende düzeni bozmamak adına bölümler arasındaki boşlukları iki olarak yaptım.

Yazılım geliştirme kendi içerisinde tasarımı barındırıyor doğru ([Kaynak](https://tr.wikipedia.org/wiki/Yaz%C4%B1l%C4%B1m_geli%C5%9Ftirme_s%C3%BCreci)), fakat burada bahsetmek istediğim yazılımda ki tasarım süreci ([Kaynak](https://tr.wikipedia.org/wiki/Yaz%C4%B1l%C4%B1m_tasar%C4%B1m%C4%B1)) değil, insanların/firmaların/şirketlerin üzerine ayrıca kafa yordukları ve kullanıcıya daha iyi bir deneyim sunabilmek için üzerinde çalıştıkları tasarım(UI&UX) disiplinlerinden bahsediyorum.

Saygılarımla.